runbook.md

```diff
@@ -1,114 +1,150 @@
-# clone repository and conda 
+## Setting up an EC2 instance
+
+Launch an AWS EC2 instance with Ubuntu 20.04 (this avoids potential troubles) with enough disk storage. 
+We need at least 500GB to download and upload datasets.
+
+Create an IAM role, called `SagemakerAdmin`, which has the following policies. 
+- `AmazonEC2ContainerRegistryFullAccess`
+- `AmazonSageMakerFullAccess`
+- `AmazonS3FullAccess`
+
+For the `SagemakerAdmin` role, make sure the Trust relationships is like following
+to grant SageMaker principal permissions to assume the role:
+
+```angular2html
+{
+    "Version": "2012-10-17",
+    "Statement": [
+        {
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "ec2.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole"
+        },
+        {
+            "Sid": "",
+            "Effect": "Allow",
+            "Principal": {
+                "Service": "sagemaker.amazonaws.com"
+            },
+            "Action": "sts:AssumeRole"
+        }
+    ]
+}
+```
+Attach the IAM role `SagemakerAdmin` to the instance. One could do so by doing:
+
+Actions -> Security ->  Modify IAM role -> Select `SagemakerAdmin` -> Update IAM role.
+
+
+### Config AWS CLI
+Before run schedule, we must config the aws cli (only region is needed, they others can be left empty).
```

Comment:
I try to config only the region and leave the others empty, but the authentication didn't work

Reply:
It seems working for Oleksandr, will resolve this.